ldes-server-port-ingest-rest/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>be.vlaanderen.informatievlaanderen.vsds</groupId>
+        <artifactId>ldes-server</artifactId>
+        <version>0.0.9-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>ldes-server-port-ingest-rest</artifactId>
+
+
+    <properties>
+        <relative-formatter-path>..</relative-formatter-path>
+    </properties>
+
+    <dependencies>
+        <!-- VSDS -->
+        <dependency>
+            <groupId>be.vlaanderen.informatievlaanderen.vsds</groupId>
+            <artifactId>ldes-server-domain</artifactId>
```

Comment:
This is still used for for configuration (appconfig) and some util functions. We have VSDSPUB-662 to figure this out.